Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```diff
@@ -285,7 +285,7 @@
     fiberColor: fibers-black
 
 - type: entity
-  parent: ClothingHandsGlovesCombat
+  parent: [ ClothingHandsGlovesColorBlack, BaseSecurityCargoContraband ]
```

Comment:
Is the reparenting to ClothingHandsGlovesColorBlack intended?
This makes them no longer be insultated even though they still have insulative fibers.

Reply:
Ah good point, fixed.